app/src/main/java/com/simformsolutions/sample/app/di/ApiModule.kt

```diff
@@ -88,6 +89,7 @@ object ApiModule {
         .create(ApiService::class.java)
 
     @Provides
+    @Named(APOLLO_CLIENT)
```

Comment:
We need `Named` when there is multiple providers for same data type. In this case `ApolloClient` is only this one so we don't need `Named` here as of now as it's giving wrong hint that there could be other `ApolloClient` instances as well with other name.